src/hammer-vlsi/hammer_vlsi/hammer_tool.py

```diff
@@ -934,6 +934,13 @@ def get_mmmc_corners(self) -> List[MMMCCorner]:
             output.append(corn)
         return output
 
+    def get_stackup(self) -> "Stackup":
+        """
+        Get the stackup provided by the technology key
+        """
+        # TODO how does python cache this? Do we need to avoid re-processing this every time?
```

Comment:
I think we can memoize this if it becomes a performance issue

Reply:
OK, I'll leave the comment in. I doubt it will be an issue.